alerts/monquorum.libsonnet

```diff
@@ -20,6 +20,19 @@
               severity_level: 'error',
             },
           },
+          {
+            alert: 'CephMonHighNumberOfLeaderChanges',
+            expr: |||
+              rate(ceph_mon_num_elections{%(cephExporterSelector)s}[15m]) > 3
+            ||| % $._config,
+            'for': $._config.monQuorumLeaderChangesAlertTime,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
```

Comment:
Please update the annotations according to :

`
annotations: {
              message: '',
              description: '',
              storage_type: '',
              severity_level: '',
            }
`

Reply:
I've created an issue to discuss this https://github.com/ceph/ceph-mixins/issues/54

but for now I'll refactor this to your suggestion.